recipes-app/iot2050setup/files/iot2050setup.py

```diff
@@ -26,7 +26,7 @@ class ansicolors:
 class TopMenu:
     def __init__(self):
         self.gscreen = SnackScreen()
-        self.boardType = subprocess.check_output('grep -a -o -P "IOT2050 .*" /proc/device-tree/model',
+        self.boardType = subprocess.check_output('grep -a -o -P "IOT2050 [\w\s]+" /proc/device-tree/model',
```

Comment:
The \s also covers the preceding space as well I think.

Reply:
yes, the space could be deleted